plugins/inputs/procstat/README.md

```diff
@@ -171,11 +200,53 @@ the `win_perf_counters` input plugin as a more mature alternative.
     - running (int)
     - result_code (int, success = 0, lookup_error = 1)
 
+If ``tcp_stats`` enabled, added fields:
+
+- procstat
```

Comment:
Shouldn't this metric be named `procstat_tcp`? We can then also remove the `tcp_` prefix from the field names...